assertion/function/assertiontree/preprocess_blocks.go

```diff
@@ -85,7 +85,9 @@ func copyGraph(graph *cfg.CFG) *cfg.CFG {
 	// must have already been copied.
 	for i, newBlock := range newGraph.Blocks {
 		for _, succ := range graph.Blocks[i].Succs {
-			newBlock.Succs = append(newBlock.Succs, copiedBlocks[succ])
+			if c, ok := copiedBlocks[succ]; ok {
+				newBlock.Succs = append(newBlock.Succs, c)
+			}
```

Comment:
Fix applied for a NilAway reported error, which seems to be a valid one. Not sure why it did not get reported earlier, though.